File: task-filter/java/src/test/java/study/gradle/taskfilterjava/HelloServiceTest.java
package study.gradle.taskfilterjava;

import static org.junit.jupiter.api.Assertions.*;

import org.assertj.core.api.Assertions;
import org.junit.jupiter.api.Test;

class HelloServiceTest {

  @Test
  void greeting() {
    var sut = new HelloService();
    var result = sut.greeting();

    Assertions.assertThat(result).isEqualTo("hello");
  }
}

File: task-filter/kotlin/src/test/kotlin/study/gradle/taskfilterkotlin/HelloServiceTest.kt
package study.gradle.taskfilterkotlin

import org.assertj.core.api.Assertions
import org.junit.jupiter.api.Test

internal class HelloServiceTest {

    @Test
    fun greeting() {
        val sut = HelloService()

        val result = sut.greeting()

        Assertions.assertThat(result).isEqualTo("hello")
    }
}